app/assets/stylesheets/hyrax/_work-show.scss

```diff
@@ -18,7 +18,12 @@ header > h1 .label {
   padding: $panel-body-padding;
 }
 
-.relationships,
+.attribute-filename,
+header h1,
+td:first-child {
```

Comment:
I think this is too generic of a selector for this.  Can we make this a new class like `.ensure-wrapped`.

Slightly off topic, but I like how github handles situations like this:
<img width="1211" alt="screen shot 2018-10-17 at 11 00 33 am" src="https://user-images.githubusercontent.com/92044/47100096-5900f980-d1fc-11e8-825d-1322d4350d63.png">

Reply:
@jcoyne - The pages have been updated to use the new class .ensure-wrapped.  Thanks for suggestion.